test/SILOptimizer/definite_init_moveonly.sil

```diff
@@ -0,0 +1,40 @@
+// RUN: %target-sil-opt -module-name definite_init_moveonly -enable-sil-verify-all -definite-init %s | %FileCheck %s
```

Comment:
did you look into why the verifier did not flag the double destroy before this change? is that a separate problem?